tests/playbooks/tests_team_exists.yml

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Please adjust the name to something like "tests_team_plugin_installation.yml" and the wireless test as well.

Reply:
updated, thanks

Reply:
Hi Till, the CI failed because the following two line are too long(> 80 char):

./tests/tests_wireless_plugin_installation_nm.yml
  6:81      error    line too long (91 > 80 characters)  (line-length)

./tests/tests_team_plugin_installation_nm.yml
  6:81      error    line too long (87 > 80 characters)  (line-length)
```
  name: Run playbook 'playbooks/tests_wireless_plugin_installation.yml' with nm as provider
  name: Run playbook 'playbooks/tests_team_plugin_installation.yml' with nm as provider
```

Reply:
@tyll , shall we change the file name to be the previous name, like "tests_team_exists.yml" ?

Reply:
diable the CI check for these two files

Reply:
It would be best to disable the line length checks for the generated files or at least those lines. Another option would be to break the lines but that might be too complicated.

Reply:
@tyll , updated the commit